primefaces/src/main/resources/META-INF/resources/primefaces/texteditor/1-texteditor-theme.css

```diff
@@ -840,7 +443,7 @@
   font-size: 32px;
 }
 .ql-snow .ql-color-picker.ql-background .ql-picker-item {
-  background-color: #fff;
+  background-color: var(--surface-a);
```

Comment:
Should this be changed? Seems like this on a fixed black background based on the rule below.

Reply:
Good catch. I reverted that one.